test/common/types.go

```diff
@@ -21,6 +21,9 @@ var (
 	RHMIOperatorNamespace             = NamespacePrefix + "operator"
 	MonitoringOperatorNamespace       = NamespacePrefix + "middleware-monitoring-operator"
 	MonitoringFederateNamespace       = NamespacePrefix + "middleware-monitoring-federate"
+	ObservabilityOperatorNamespace    = NamespacePrefix + "observability-operator"
+	ObservabilityProductNamespace     = NamespacePrefix + "observability"
+	ObservabilityNamespacePrefix      = NamespacePrefix + "observability-"
```

Comment:
Hey @valerymo wdyt if we would remove the observabilityNamespacePrefix and instead use the observabilityProductNamespace? We would add the `-` at each string where it's required. 
Not a big deal if you would like this to stay as it is, however, namespacePrefix seems more like "redhat-rhoam" or "redhat-sandbox" etc rather than adding a `-` to ProductNs.

Reply:
Hey @MStokluska , thank you very much for your comment. I also thought about this - add "-" to code at each place or just define this as prefix.  But main thing, as seems to me, code maintenance;   "-" at 34  places - it's a source of mistakes. Also, there are another definitions in alerts_exist.go : File: NamespacePrefix + "...." , so I wanted to do similar: File: ObservabilityNamespacePrefix + "..." .  The issue that NamespacePrefix is not "redhat-rhoam", it's "redhat-rhoam-" (defined at GetNamespacePrefix()) 
We can see also "statistics" : ) here:
[common (vm_MGDAPI-1882 *)]$ grep "File: ObservabilityNamespacePrefix +" alerts_exist.go |wc -l
34
[common (vm_MGDAPI-1882 *)]$ grep "File: NamespacePrefix +" alerts_exist.go |wc -l
37
[common (vm_MGDAPI-1882 *)]$ grep "File: NamespacePrefix + \"-" alerts_exist.go |wc -l
0
But agree, that in types.go file it looks a bit not usual. Seems to me - two ways 1) return ObservabilityNamespacePrefix back to file. 2nd option - leave it as is (...maybe move it to the end of the var list, to "separate" from namespaces ... ) 
Thank you very much for your comment!